CITATION.cff

```diff
@@ -0,0 +1,19 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Leaflet
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Volodymyr
+    family-names: Agafonkin
+    email: agafonkin@gmail.com
+repository-code: 'https://github.com/Leaflet/Leaflet'
+url: 'https://leafletjs.com'
+license: BSD-2-Clause
+commit: d15112c
+version: 1.9.4
+date-released: '2023-05-18'
```

Comment:
I think we should omit the `commit` / `version` / `date-released` fields since they are optional and we don't want to have the hassle of updating them constantly.

```suggestion
```

Reply:
agreed